product_unique_serial/models/stock_move.py

```diff
@@ -52,26 +69,23 @@ def check_unicity_move_qty(self):
                         ) % (move.product_id.name))
 
     @api.model
-    def check_unicity_qty_available(self, operation_or_move, lot_id):
+    def check_unicity_qty_available(self, operation_or_move):
         """
-        Check quantity on hand to verify that has qty = 1
+        Check quantity on lot to verify that has qty = 1
         if 'lot unique' is ok on product
         """
+        lot_id = operation_or_move.lot_id
```

Comment:
Done. I check all test in local and all is correct, except test 4, but I think that it's wrong because we can have the same product with diferent serial numbers.